cfgov/v1/jinja2/v1/_layouts/base.html

```diff
@@ -99,51 +99,10 @@
         docElement.className = docElement.className.replace( /(^|\s)no-js(\s|$)/, '$1$2' );
     </script>
 
-    {# TODO: jQuery CDN is provided in the head to satisfy
-             Google Tag Manager (GTM) requirements.
-             Ideally GTM would handle its own dependency management
-             and initializing jQuery could be moved to the footer. #}
-    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
-    <script>if (!window.jQuery) { document.write('<script src="{{ static('js/jquery.min.js') }}"><\/script>'); }
```

Comment:
Pretty sure we still need L:102 - L:108. Unless something has drastically changed, GTM is still looking for jQuery in the head https://github.com/cfpb/cfgov-refresh/issues/587